packages/session/examples/cloudflare-kv.ts

```diff
@@ -0,0 +1,51 @@
+import { env } from 'cloudflare:test'
+import { Hono } from 'hono'
+import type { SessionEnv } from '../src'
+import { useSession, useSessionStorage } from '../src'
+import * as cookies from '../src/cookies'
+
+export const secret = cookies.generateId(16)
+
+/**
+ * Example hono app using Cloudflare KV as session storage.
+ *
+ * This example assumes you have a Cloudflare KV namespace named `SESSION_KV`.
+ */
+export const app = new Hono<SessionEnv>()
+
+app.use(
+  useSessionStorage((c) => ({
+    delete(sid) {
+      c.executionCtx.waitUntil(env.SESSION_KV.delete(sid))
+    },
+    get(sid) {
+      return env.SESSION_KV.get(sid, 'json')
+    },
+    set(sid, data) {
+      c.executionCtx.waitUntil(
+        env.SESSION_KV.put(sid, JSON.stringify(data), {
+          // Optionally configure session data to expire some time after the session cookie expires.
+          expirationTtl: 2_592_000, // 30 days in seconds
+        })
+      )
+    },
+  })),
+  useSession({ secret })
+)
+
+app.get('/session', async (c) => {
```

Comment:
I updated the examples to the "not chained" style. It's common for users. If you pass `SessionEnv` to `Hono` as a generic, you can write it like this.